client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -104,35 +104,6 @@
                     </span>
                 </div>
 
-                <!-- state column -->
-                <ng-container matColumnDef="state">
-                    <mat-header-cell *matHeaderCellDef mat-sort-header>State</mat-header-cell>
-                    <mat-cell (click)="openEditInfo(motion, $event)" *matCellDef="let motion">
-                        <div class="fill">
-                            <div class="innerTable state-column">
-                                <div class="ellipsis-overflow" *ngIf="motion.category">
-                                    <os-icon-container icon="device_hub">
-                                        {{ motion.category.prefixedNameWithParents }}
```

Comment:
Did you remove all usages of `prefixedNameWithParents`? If so, please remove this property.

Reply:
Still used in selector classes